frontend/app/views/spree/checkout/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1,64 @@
+____________________________________________________________
```

Comment:
Thought about this slightly more, and it's probably preferable to keep this HTML and to  change the `textbox` into a `div` with a fixed height and `overflow: scroll`